server/apiServer.js

```diff
@@ -8,6 +8,7 @@ const express = require('express');
 const cookieParser = require('cookie-parser');
 const bodyParser = require('body-parser');
 const cors = require('cors');
+const sharetribeSdk = require('sharetribe-flex-sdk');
```

Comment:
Was this import a leftover from fixing conflicts from rebase? No need to import the SDK here anymore.

Reply:
Yeah there was quite a lot of merge conflicts so I think this is just leftover, I will remove it